pytest_tests/testsuites/services/s3_gate/test_s3_object.py

```diff
@@ -915,3 +916,17 @@ def test_s3_sync_dir(self, sync_type):
                 #     "FULL_CONTROL",
                 #     "FULL_CONTROL",
                 # ], "Permission for all groups is FULL_CONTROL"
+
+    @allure.title("Test S3 Put 10 nested level object")
+    def test_s3_put_10_folder(self):
+        path = "/".join(["".join(choices(string.ascii_letters, k=3)) for _ in range(10)])
+        file_path_1 = os.path.join(os.getcwd(), ASSETS_DIR, path, "test_file_1")
```

Comment:
I was told that we should get rid of ASSETS_DIR var, and we have preprare_tmp_dir fixture which returns 
`os.path.join(os.getcwd(), ASSETS_DIR)`